### PR TITLE
all test cases created for vehicle table view

### PR DESCRIPTION
**Pull request by esrasatiroglu**
Hello Cevriye,
You can write some comments here

### CODE REVIEW

File: configuration.properties
Comment (CevriyeTurkmen) [inline configuration.properties:1]: Looking good

**Comment by CevriyeTurkmen** [comment]
Looking good but it says there are two conflicts, lets ask Matt:)

**Comment by CevriyeTurkmen** [comment]
Looking good but it says there are two conflicts, lets ask Matt:)

**Review by CevriyeTurkmen — APPROVED** [review]
There are 2 conflicts which has to be solved

**Review by AlexPro888 — APPROVED** [review]

**Pull request merged**